src/components/CategoryPicker.tsx

```diff
@@ -21,7 +21,7 @@ type CategoryPickerProps = CategoryPickerOnyxProps & {
     /** It's used by withOnyx HOC */
     // eslint-disable-next-line react/no-unused-prop-types
     policyID: string;
-    selectedCategory: string;
+    selectedCategory: string | undefined;
```

Comment:
WDYT? If the component supports the case when the param is undefined, I guess the param is optional.
```suggestion
    selectedCategory?: string;
```